docs/pages/access-controls/access-requests/access-request-configuration.mdx

```diff
@@ -0,0 +1,622 @@
+1. Calculate the session TTL of the certificate the user would receive if
+   Teleport were to grant the Access Request. This calculation chooses the
+   lowest value of: 
+   - The requested session expiration time, which is the value of the
+     `--session-ttl` flag of [`tsh request
+     create`](../../reference/cli/tsh.mdx#tsh-request-create).
+   - The remaining time in the user's current Teleport session.
+   - The lowest value of the `options.max_session_ttl` field in the user's
+     requested roles.
+
+1. If the maximum duration is greater than zero, set the duration of elevated
+   privileges to either the maximum duration or the session TTL calculated
+   earlier, whichever is shorter. Otherwise, set the duration of elevated
+   privileges to the session TTL.
+
+### How long Access Requests are valid
+
+Teleport uses the following logic to determine how long an Access Request is
+valid while it awaits approval:
+
+1. Begin with with the base expiration of the Access Request, which a user can
+   set with the `--request-ttl` flag of the [`tsh request
+   create`](../../reference/cli/tsh.mdx#tsh-request-create) command. If this is
+   unset, the request TTL is one hour.
+
+1. If the user's Teleport session is due to expire before the base expiration
+   time, Teleport sets the Access Request expiration to the end of the Teleport
+   session.
+
+1. If any of the Teleport roles requested by the Access Request has an
+   `options.max_session_ttl` that expires before the expiration time, Teleport
+   sets the expiration of the Access Request to that time.
+
+1. Return an error if the value of `--request-ttl` is greater than the request
+   TTL calculated in the previous step.
+
+## How clients request access
+
+A user's Teleport roles determine how they submit Access Requests, whether
+Access Requests are optional or mandatory, and whether a user must provide a
+reason when making a request.
+
+A role's `options.request_access` setting specifies a strategy for producing
+Access Requests. It can include one of the following values:
+
+|Value|Meaning|
+|---|---|
+|`optional`|The default. The user does not need to specify a reason when making a request. The user must initiate a request manually when they log in to Teleport.|
+|`always`| When a user signs in to Teleport, their client automatically generates an Access Request for all roles available to the user, without providing a reason.|
+|`reason`| When a user signs in to Teleport, their client automatically generates an Access Request for all roles available to the user. The user must provide a reason when authenticating.|
+
+If a role includes the `reason` strategy, you can specify a prompt to remind the
+user to provide a reason if the user attempts to create an Access Request
+without one. To do so, set the `options.request_prompt` option in a role.
+
+For example, the following role prompts the user with the text, "Please provide
+your ticket ID":
+
+
+```yaml
+kind: role
+version: v6
```

Comment:
Would it make sense to use the same version for all the roles on this page? Otherwise people might wonder if the discrepancy is intentional..